lib/rbac/authorizer.rb

```diff
@@ -2,30 +2,23 @@ module Rbac
   class Authorizer
     include Vmdb::Logging
 
-    def self.role_allows?(*args)
-      new.role_allows?(*args)
+    def self.role_allows?(**args)
+      new.role_allows?(**args)
     end
 
-    def role_allows?(options = {})
-      user    = options[:user]
+    # @options option any         - any of the identifiers can match (default: false)
+    # @options option identifier  - comes from the back end (ex: User#role_allows?)
+    # @options option feature     - same as identifier, comes from the front end (ex: ApplicationHelper#role_allows?)
+    # @options option identifiers - same as others. deprecated
```

Comment:
If identifiers is deprecated should we add a deprecated warning?

Reply:
There was a comment that we only use this in 2 places in the ui.
I have a dependent PR that removes this from there.

But for `UserRole#allows_any?()` we still accept `:identifiers` so I am unsure if we want to deprecate it in one place and then require it in another.

Reply:
I'm just going to let this be.

I do have in the code that will allow multiple features to come across as `feature: []` in addition to `identifiers: []` so if in the future we decide to limit this api, we can do so.

I was trying to close what I read as a TODO item, but am not totally sold on the merrit.